components/NewRealmWizard/components/steps/CommunityTokenDetailsForm.tsx

```diff
@@ -95,7 +92,11 @@ export default function CommunityTokenForm({
     setValue,
     handleSubmit,
     formState: { isValid },
-  } = useForm({
+  } = useForm<any>({
+    // @asktree: I set default values here in order to eliminate a bug where a value was only being set as a side effect of opening advanced options
+    defaultValues: {
+      useSupplyFactor: true,
+    },
```

Comment:
I think essentially everything should have a default value and it should be set here. However due to the way this component is made many of the default values will be immediatelly overwritten or ignored, so adding them here would just be confusing. I don't think refactoring the component to be what I consider idiomatic would be a good use of time, though.

Reply:
That said maybe it would improve some behavior or something to add this here, I didn't check. I only added `useSupplyFactor` because it fixes an actual user-facing bug.